addons/api/mirage/serializers/target.js

```diff
@@ -17,14 +17,14 @@ export default ApplicationSerializer.extend({
     );
     const { hostSets } = this.schema;
     if (model.hostSetIds?.length) {
```

Comment:
hostSourceIds

Reply:
The mock relationship is monomorphic for now.